content/coronavirus_landing_page.yml

```diff
@@ -1,7 +1,7 @@
 content:
-  title: "Coronavirus (COVID-19): what you need to do"
-  meta_description: "Find out about the government response to coronavirus (COVID-19) and what you need to do."
-  page_header: "Coronavirus (COVID-19): what you need to do"
+  title: "Coronavirus (COVID-19): guidance and support"
+  meta_description: "Find information on coronavirus, including guidance, support, announcements and statistics."
+  page_header: "Coronavirus (COVID&#8209;19)"
```

Comment:
Why change the dash? Have you tested this? I'm a bit nervous it'll render it as the code and not the symbol - if it worked before, I'd suggest leaving it as it was.

Reply:
> I'm a bit nervous it'll render it as the code and not the symbol 

@andysellick Yes you are absolutely right, it would render as the code and not the symbol with how things are currently. As stated in the desc of this PR, that is why I opened this other PR in collections to support this change https://github.com/alphagov/collections-private/pull/3 😁 

> If it worked before, I'd suggest leaving it as it was.

So it did work before, however at a certain breakpoint **COVID-19** will wrap onto 2 lines. 
![www gov uk_coronavirus(iPhone_X)](https://user-images.githubusercontent.com/7116819/81426375-02963b80-9151-11ea-9db6-b242c63b8e4c.png)

It is part of the checklist in Trello that we remedy this linebreak issue along with the content changes (which this does by using the special non-breaking hyphen character). 

I know this is ugly and hacky, and I'm sure there are other more elegant solutions for this issue, so please do correct me if you can think of a better way.

Or alternatively we could revert to using a "normal" hyphen and implement a more robust solution for the wrapping issue later on (as long as Sam & everyone are okay with that)?

Reply:
Ah, got it. Sorry, hadn't read the description properly.

Reply:
No problem, I'm sorry I got kind of snarky there 🤦‍♀️